wpilibc/src/main/native/include/frc/simulation/SolenoidSim.h

```diff
@@ -16,7 +16,7 @@ class SolenoidSim {
   SolenoidSim(std::shared_ptr<PneumaticsBaseSim> moduleSim, int channel);
   SolenoidSim(int module, PneumaticsModuleType type, int channel);
   SolenoidSim(PneumaticsModuleType type, int channel);
-  virtual ~SolenoidSim() = default;
+  ~SolenoidSim() = default;
```

Comment:
This is needed if there are any derived classes of SolenoidSim.  Are there or would there be any reason to be?

Reply:
Not that I know of. It looks like it was only added because `RegisterOutputCallback` being virtual was causing a "virtual functions but no virtual destructors" warning. `RegisterOutputCallback` looks like it was virtual because it was copied and pasted from somewhere else